tables/basic-table/step5.md

```diff
@@ -0,0 +1,120 @@
+In this step, we'll extend out dataset and move it into an external file so our component file stays nice and simple.
+
+## Task
+
+1) Move your row/column definitions to an external file
+
+<pre class="file" data-filename="src/data.js" data-target="replace">
+export const columns = [
+  { title: "First column" },
+  { title: "Second column" },
+  { title: "Third column" }
+];
+export const defaultRows = [
+  {
+    cells: [
+      { title: "Row 1 column 1" },
+      { title: "Row 1 column 2" },
+      { title: "Row 1 column 3" }
+    ]
+  },
+  {
+    cells: [
+      { title: "Row 2 column 1" },
+      { title: "Row 2 column 2" },
+      { title: "Row 2 column 3" }
+    ]
+  },
+  {
+    cells: [
+      { title: "Row 3 column 1" },
+      { title: "Row 3 column 2" },
+      { title: "Row 3 column 3" }
+    ]
+  }
+];
+</pre>
+
+2) Add three more same-type rows to your rows definition, and update the text within them to reflect which row they belong to.
+
+<pre class="file" data-filename="src/data.js" data-target="replace">
```

Comment:
Thinking we should only show the new rows here. And make this a copy Vs replace.